src/index.js

```diff
@@ -1,4 +1,9 @@
-require('es6-promise').polyfill();  // Polyfill `Promise`.
+// Polyfill `Promise`.
+var promise = require('promise-polyfill');
+if (!global.Promise) {
```

Comment:
I don't think that it's even necessary. 
```javascript
require('promise-polyfill');
````
it should be enough. It seems the polyfill already attaches itself to the global scope if Promise is not defined:

https://github.com/taylorhakes/promise-polyfill/blob/master/Promise.js#L196